recipes/genutil/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "genutil" %}
+{% set version = "2.8.2" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    url: https://github.com/UV-CDAT/genutil/archive/{{ version }}.tar.gz
```

Comment:
I guess that the `fn:` field is no longer mandatory. I just learned a new thing :smile: